<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supprimer_Billets</class>
 <widget class="QDialog" name="Supprimer_Billets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Supprimer Billets</string>
   </property>
   <widget class="QPushButton" name="supp_billets_select">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/g);</string>
    </property>
    <property name="text">
     <string>Supprimer billet par id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supp_billet_nonvalable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/g);</string>
    </property>
    <property name="text">
     <string>Supprimer toutes les billets non valable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/g);</string>
    </property>
    <property name="text">
     <string>retour</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>430</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ID de billet a supprimer</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-15</y>
      <width>621</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>supp_billets_select</zorder>
   <zorder>supp_billet_nonvalable</zorder>
   <zorder>retour</zorder>
   <zorder>lineEdit</zorder>
   <zorder>tableView</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>retour</sender>
   <signal>clicked()</signal>
   <receiver>Supprimer_Billets</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
